scripts/kind-e2e/e2e.sh

```diff
@@ -328,13 +392,15 @@ mkdir -p ${PRJ_ROOT}/output/kind-config/dapper/ ${PRJ_ROOT}/output/kind-config/l
 SUBMARINER_BROKER_NS=submariner-k8s-broker
 SUBMARINER_PSK=$(cat /dev/urandom | LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 64 | head -n 1)
 KUBEFED_NS=kube-federation-system
+OLM_VERSION=0.11.0
 export KUBECONFIG=$(echo ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster{1..3} | sed 's/ /:/g')
 
 kind_clusters "$@"
 setup_custom_cni
 if [[ $3 = true ]]; then
     enable_logging
 fi
+#install_olm
```

Comment:
This is commented, it won't install, right?

Reply:
> This is commented, it won't install, right?

Yes, Daniel and I thought it best to leave it out for the time being, since nothing needs it yet. To answer your Slack question, it seems we do need OLM to deploy the operator, at least as it’s supposed to be packaged for OpenShift (where OLM takes care of CRD dependencies etc.).

Reply:
Ok, althought that would be more midstream/downstream testing I guess.